### ISSUE TITLE
CTSM-FATES v1 compsets

### ISSUE THREAD

**Issue by wwieder** [ESCOMP/CTSM#1047]
Create _working_ compsets for land only (I) spin-up, historical & SPP's with and w/out active crops.  Also, F2000 and F_AMIP for coupled model output.

Would the compsets also define if we're using FATES-sp, FATES-fixed biogeography, FATES-dynamic biogeography. 

This is linked with #1046.

**Comment by ekluzek** [comment]
This requires #1046 in order to move forward.

**Comment by ekluzek** [comment]
The F compsets will need to go into CAM. So this will require a CTSM tag, CAM tag, and CESM tag to put them all together.

**Comment by ekluzek** [comment]
These are some notes from @rosiealice on what she thought was important...

> For the moment, I think the important ones to get us going are
> 1850
> 2000
> Transient
> SSP -something.
> 
> -As you note,  FATES-CROP will need more coding work, as will FATES-SP (that's on me).  
> -I don't think that having CLM4.5 compsets is a big priority? Does anyone think otherwise?  
> 
> -On the other details, it would be nice to have a version which is the fastest possible version - i.e. with NO glaciers, runoff, crops, etc,  and then one with something more like a default configuration. People who want to run FATES with a specific river model are quite rare, I suspect. 
> 
> -The no-anthro compset is mostly reelevant to the situation where we have crops working, right? Maybe we can leave that until later?  
> 
> -I guess the other thing is climate forcing data? Doing that rather depends on whether we want to create the whole entire universe of FATES compsets in one go? I don't have a good handle on the implications for e.g. testing, of adding a massive number of compsets, so if people have thoughts on that please chime in...

**Comment by ekluzek** [comment]
We don't have it as a compset yet, but ctsm5.1.dev056 enables the ability to do FATES-SP mode.  And we also have the ability to have fixed- biogeography. So this brings in much of the functionality needed for FATES v1.

**Comment by ekluzek** [comment]
As part of #1609 we'll be adding a single-point compset for FATES I1PtClm51FatesRs, which gives one of the needed compsets for FATESv1 (although it wasn't mentioned above it's still one of the needed ones).

**Comment by ekluzek** [comment]
#1617 is about adding transient FATES compset aliases.

**Comment by rosiealice** [comment]
I think we have fates compsets now, and the transient compsets are their own issue.

**Comment by ekluzek** [comment]
Do we want this for cesm3.0?

**Comment by wwieder** [comment]
Yes, for FATES sp and fixed biogeography configurations in I cases

**Comment by ekluzek** [comment]
We can do all of these configurations. I'm thinking definition of done here is to have compset aliases and long-names that make it easy to do all of these options. @wwieder and @rosiealice does that sound right to you?

Is fixed-biogeography the same as NOCOMP from the CESM3 compsets for CTSM?

https://docs.google.com/spreadsheets/d/1nJl4lbbwBq60quyaERJ6OvVJxF4kDu4xX7zZenZgyB8

**Comment by wwieder** [comment]
Let's discuss at the SE meeting, but I'm not sure these will necessarily all be possible. I also don't know if any need aliases (which at last check were being reserved for select, scientifically supported model configurations in the CESM3.0 release).
